src/extension/devtools/devtools.ts

```diff
@@ -98,14 +98,15 @@ function startRequestInterval(ms = 500) {
         payload: await rpcClient.request("getClientOperations"),
       });
     }
+
+    id = setTimeout(() => getClientData(), ms);
```

Comment:
```suggestion
    id = setTimeout(getClientData, ms);
```
I think we don't need the function definition here

Reply:
Also, should we wrap the whole block in try..finally to ensure an exception doesn't break the loop?

Reply:
Thats a good idea!